contracts/core/extensions/CoreAccounting.sol

```diff
@@ -74,7 +74,10 @@ contract CoreAccounting is
         external
         nonReentrant
     {
+        // Call internal withdraw function
         withdrawInternal(
+            msg.sender,
+            msg.sender,
```

Comment:
Pass `msg.sender` to `withdrawInternal` just like with `depositInternal`

Reply:
When is it supposed to be `_from` and `_to` being different?

Reply:
I don't think there's a use case where they're different right now, thoughts? Added it here to mirror how it's done during `depositInternal`